v2/stats/metrics.proto

```diff
@@ -63,6 +63,7 @@ message MemoryStat {
 	uint64 usage_limit = 33;
 	uint64 swap_usage = 34;
 	uint64 swap_limit = 35;
+	uint64 fail_cnt = 36;
```

Comment:
field name should correspond to memory.events file

Reply:
Not sure that, `max` will be explanatory enough.

Reply:
events_max?